mbf_costmap_nav/src/mbf_costmap_nav/free_pose_search.cpp

```diff
@@ -329,7 +346,16 @@ SearchSolution FreePoseSearch::search() const
       in_queue_or_visited.insert(cell_index);
       queue.push(neighbor);
     }
-  }
+
+    if (queue.empty())
+    {
+      break;
+    }
+
+    test_cell = queue.top();
+    queue.pop();
+    costmap2d->mapToWorld(test_cell.x, test_cell.y, sol.pose.x, sol.pose.y);
```

Comment:
Doesn't this line overwrite the `outside_or_unknown` solution candidate saved [here](https://github.com/magazino/move_base_flex/pull/327/files?diff=unified&w=0#diff-43fb1b9fdaf229fde2a9661293a0da41b5bfba67e7843bdb79dd19ce01fdd3cbR309)?

Reply:
you are correct. Fixed